billing/views.py

```diff
@@ -128,15 +128,31 @@ def subscription_schedule_updated(
                 ),
             )
 
-    def subscription_schedule_released(self, schedule: stripe.Subscription) -> None:
+    def subscription_schedule_released(self, schedule: stripe.SubscriptionSchedule) -> None:
```

Comment:
I have manually tested all of these handlers with the mock Stripe API, with the exception of this one, which I couldn't figure out how to get to fire. Test cases seem to imply it works though.

Reply:
Update: I triggered this accidentally, and it seems to have worked - not sure how to do it again though 😆